NEW MECH REBALANCES

ScarletLightweaverSS14 opened the pull request:
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
Rebalance mechs to make them a viable option

## Why we need to add this
right now mechs are pretty shit. i think we call can agree on that

## Media (Video/Screenshots)
NA

## Checks
<!-- check boxes for faster reviewing of your PR -->

- [ ] I do not require assistance to complete the PR.
- [x] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
NEW MECH REBALANCE

:cl: SCARLET LIGHTWEAVER
- tweak: Changed AC-2 from burst to full auto also changed fire rate from 5 to 3.2.
- tweak: Changed AC-10 from semi to full auto.
- tweak: Changed Gygax speed to faster(from: walk 2.4 and sprint 3.7 to: 2.5 and 4.0) and armor.
- tweak: Changed Durand armor and HP from 400 to 465 also changed pilot damage multiplier from 25% to 15%.
- tweak: Changed Marauder HP from 500 to 575.
- tweak: Changed Seraph HP from 550 to 700.
- tweak: Changed DarkGygax HP from 300 to 375 and speed from 2.2, 4.0 to 3.0, 4.5.
- tweak: Changed Mauler armor and HP from 500 to 550, now mauler has point light.

ScarletLightweaverSS14 committed:
NEW MECH REBALANCES

ScarletLightweaverSS14 committed:
bug fixes

ScarletLightweaverSS14 committed:
tweak on hades

ScarletLightweaverSS14 committed:
bug fixes

Rinary1 commented on Content.Server/_Starlight/Mech/MechBatteryWhitelistSystem.cs:
Tbh i don't see any reason to add this.

Rinary1 commented on Resources/Locale/en-US/mech/mech.ftl:
```diff
@@ -20,4 +20,6 @@ mech-no-enter = You cannot pilot this.
 
 mech-eject-pilot-alert = {$user} is pulling the pilot out of the {$item}!
 
+mech-power-cage-too-large = That power cage won't fit inside!
```
Tbh i don't see any reason to add this.

Rinary1 commented on Resources/Prototypes/Damage/modifier_sets.yml:
```diff
@@ -389,6 +389,18 @@
+    Heat: 0.8
+  flatReductions:
+    Blunt: 3
+    Heat: 3
```
It is not worth simply adding new modifier sets. The point was that everyone had the same characteristics(I mean between light armors) because they had “the same armor plates.”

Rinary1 commented on Resources/Prototypes/Damage/modifier_sets.yml:
```diff
@@ -436,3 +448,27 @@
+    Heat: 0.7
+  flatReductions:
+    Blunt: 10
+    Heat: 8
```
It is not worth simply adding new modifier sets. The point was that everyone had the same characteristics(I mean between light armors) because they had “the same armor plates.”

Rinary1 commented on Resources/Prototypes/Entities/Objects/Power/powercells.yml:
```diff
@@ -109,11 +110,14 @@
   components:
+  - type: Tag
+    tags:
+      - SmallPowerCell
```
medium powercell... Small PoweCell?

Rinary1 commented on Resources/Prototypes/Entities/Objects/Power/powercells.yml:
```diff
@@ -144,11 +148,14 @@
   components:
+  - type: Tag
+    tags:
+      - SmallPowerCell
```
hight powercell... Small PoweCell?

Rinary1 commented on Resources/Prototypes/Entities/Objects/Power/powercells.yml:
```diff
@@ -184,6 +191,9 @@
   components:
+  - type: Tag
+    tags:
+      - SmallPowerCell
```
hyper powercell... Small PoweCell?

Rinary1 commented on Resources/Prototypes/Entities/Objects/Specific/Mech/Weapons/Gun/combat.yml:
```diff
@@ -265,12 +265,10 @@
-    shotsPerBurst: 3
-    burstCooldown: 1.2
-    selectedMode: Burst
+    fireRate: 4.3
```
make fire rate smaller if you making it full auto.

Rinary1 commented on Resources/Prototypes/tags.yml:
```diff
@@ -1236,12 +1233,24 @@
 - type: Tag
   id: PowerCage
 
+- type: Tag
+  id: PowerCageHigh
+
+- type: Tag
+  id: PowerCageMedium
+
+- type: Tag
+  id: PowerCageSmall
+
```
move this to _Starlight actually?

Rinary1 commented on Resources/Prototypes/tags.yml:
```diff
@@ -1236,12 +1233,24 @@
 - type: Tag
   id: PowerCellSmall
 
+- type: Tag
+  id: SmallPowerCell
+
```
move this to _Starlight actually?

Rinary1 commented on Resources/Prototypes/Entities/Objects/Specific/Mech/mechs.yml:
Any starlight comments?

ScarletLightweaverSS14 commented:
@Rinary1 
We NEED the C# system because base game doesn't validate ItemSlots for mech batteries
We NEED the MechSystem edit because it doesn't respect args.Handled
We NEED tags because YAML can't distinguish battery types otherwise
The actual _Starlight custom content (damage modifiers, HADES changes) is just balance/content additions

Rinary1 commented:
>  gygax are actually lightly armored mechs now. it can no longer use medium/high cap power cages!

Tbh i don't see any reason to do this.

> @Rinary1 We NEED the C# system because base game doesn't validate ItemSlots for mech batteries We NEED the MechSystem edit because it doesn't respect args.Handled We NEED tags because YAML can't distinguish battery types otherwise The actual _Starlight custom content (damage modifiers, HADES changes) is just balance/content additions

Okay, I'll say in another words:

Why we need to validate item sloots for mech batteries? 

I'm about this:

> gygax are actually lightly armored mechs now. it can no longer use medium/high cap power cages!

I don't see any points why this nerf should be added. And why this added only for gygax(if it not only for gygax then your changelog is incorrect)

Rinary1 commented:
I don't write anything about tags and mech system

ScarletLightweaverSS14 commented:
well @Rinary1  its to make up for the speed of the gygaxes them having the ability to have medium and large power cages is very op

> > gygax are actually lightly armored mechs now. it can no longer use medium/high cap power cages!
> 
> Tbh i don't see any reason to do this.
> 
> > @Rinary1 We NEED the C# system because base game doesn't validate ItemSlots for mech batteries We NEED the MechSystem edit because it doesn't respect args.Handled We NEED tags because YAML can't distinguish battery types otherwise The actual _Starlight custom content (damage modifiers, HADES changes) is just balance/content additions
> 
> Okay, I'll say in another words:
> 
> Why we need to validate item sloots for mech batteries?
> 
> I'm about this:
> 
> > gygax are actually lightly armored mechs now. it can no longer use medium/high cap power cages!
> 
> I don't see any points why this nerf should be added. And why this added only for gygax(if it not only for gygax then your changelog is incorrect)

ScarletLightweaverSS14 commented:
Medium/high power cages would give it too much uptime, breaking the intended playstyle

ScarletLightweaverSS14 commented:
if you REALLY want- i cant remove it but like im afraid it might become the meta for mechs then @Rinary1

ScarletLightweaverSS14 committed:
changes as per @1Rinary's request

Rinary1 commented:
> Medium/high power cages would give it too much uptime, breaking the intended playstyle

Who establishes your "intended playstyle"?
So, shall we start nerfing agents and nukeops so that they only play covertly?
We shoudn't nerf things because someone think that players using "uninteded playstyle".

ScarletLightweaverSS14 commented:
> > Medium/high power cages would give it too much uptime, breaking the intended playstyle
> 
> Who establishes your "intended playstyle"? So, shall we start nerfing agents and nukeops so that they only play covertly? We shoudn't nerf things because someone think that players using "uninteded playstyle".

this isnt a nerf. they can still use norma powercells just fine. this is a rebalance for the gygax it trades power capacity and armor for speed

Rinary1 commented:
> > > Medium/high power cages would give it too much uptime, breaking the intended playstyle
> > 
> > 
> > Who establishes your "intended playstyle"? So, shall we start nerfing agents and nukeops so that they only play covertly? We shoudn't nerf things because someone think that players using "uninteded playstyle".
> 
> this isnt a nerf. they can still use norma powercells just fine. this is a rebalance for the gygax it trades power capacity and armor for speed

speed shoudn't be compensated by powercell size. It just doesn't make sense.

ScarletLightweaverSS14 commented:
> > > > Medium/high power cages would give it too much uptime, breaking the intended playstyle
> > > 
> > > 
> > > Who establishes your "intended playstyle"? So, shall we start nerfing agents and nukeops so that they only play covertly? We shoudn't nerf things because someone think that players using "uninteded playstyle".
> > 
> > 
> > this isnt a nerf. they can still use norma powercells just fine. this is a rebalance for the gygax it trades power capacity and armor for speed
> 
> speed shoudn't be compensated by powercell size. It just doesn't make sense.

how so?

Rinary1 commented:
> how so?

because actually you can insert reactor or antique? It's also op? Your logic is strange, if you think that speed is too much op, then don't buff it??? You just making op thing and nerf another which not related at all to balance it... It's strange.

ScarletLightweaverSS14 commented:
PowerCageSmall: Tier 2 Arsenal (BasicShuttleArmament tech, 10,500 points)
PowerCageMedium: Tier 2 Arsenal (BasicShuttleArmament tech, 10,500 points)
PowerCageHigh: Tier 3 Arsenal (AdvancedShuttleWeapon tech, 15,000 points)

so if sci isnt shit they can easily unlock this

Rinary1 commented:
a

> PowerCageSmall: Tier 2 Arsenal (BasicShuttleArmament tech, 10,500 points) PowerCageMedium: Tier 2 Arsenal (BasicShuttleArmament tech, 10,500 points) PowerCageHigh: Tier 3 Arsenal (AdvancedShuttleWeapon tech, 15,000 points)
> 
> so if sci isnt shit they can easily unlock this

reactor also tier 3 they can easiliy unlock it.

ScarletLightweaverSS14 commented:
> > how so?
> 
> because actually you can insert reactor or antique? It's also op? Your logic is strange, if you think that speed is too much op, then don't buff it??? You just making op thing and nerf another which not related at all to balance it... It's strange.

isnt  the powerdraw too great to actually bring value? plus the antique is very rare and hard to find

Rinary1 commented:
a

> > > how so?
> > 
> > 
> > because actually you can insert reactor or antique? It's also op? Your logic is strange, if you think that speed is too much op, then don't buff it??? You just making op thing and nerf another which not related at all to balance it... It's strange.
> 
> isnt the powerdraw too great to actually bring value? plus the antique is very rare and hard to find

move fast -> enter in maints -> wait 10-20 seconds -> move fast. It's more op than just power cage because you need recharge it with recharger when reactor it's just wait

ScarletLightweaverSS14 commented:
> a
> 
> > > > how so?
> > > 
> > > 
> > > because actually you can insert reactor or antique? It's also op? Your logic is strange, if you think that speed is too much op, then don't buff it??? You just making op thing and nerf another which not related at all to balance it... It's strange.
> > 
> > 
> > isnt the powerdraw too great to actually bring value? plus the antique is very rare and hard to find
> 
> move fast -> enter in maints -> wait 10-20 seconds -> move fast. It's more op than just power cage because you need recharge it with recharger when reactor it's just wait

counter point being that you can barely use the weapons- if you miss it will be very punishing

Rinary1 commented:
> > a
> > > > > how so?
> > > > 
> > > > 
> > > > because actually you can insert reactor or antique? It's also op? Your logic is strange, if you think that speed is too much op, then don't buff it??? You just making op thing and nerf another which not related at all to balance it... It's strange.
> > > 
> > > 
> > > isnt the powerdraw too great to actually bring value? plus the antique is very rare and hard to find
> > 
> > 
> > move fast -> enter in maints -> wait 10-20 seconds -> move fast. It's more op than just power cage because you need recharge it with recharger when reactor it's just wait
> 
> counter point being that you can barely use the weapons- if you miss it will be very punishing

No? I played on mech with reactor power cell and it works well

ScarletLightweaverSS14 commented:
> > > a
> > > > > > how so?
> > > > > 
> > > > > 
> > > > > because actually you can insert reactor or antique? It's also op? Your logic is strange, if you think that speed is too much op, then don't buff it??? You just making op thing and nerf another which not related at all to balance it... It's strange.
> > > > 
> > > > 
> > > > isnt the powerdraw too great to actually bring value? plus the antique is very rare and hard to find
> > > 
> > > 
> > > move fast -> enter in maints -> wait 10-20 seconds -> move fast. It's more op than just power cage because you need recharge it with recharger when reactor it's just wait
> > 
> > 
> > counter point being that you can barely use the weapons- if you miss it will be very punishing
> 
> No? I played on mech with reactor power cell and it works well

Also rinary i think the speed buff is needed considering its said to be ''lightly'' armored and right now it moves kind of slow

Rinary1 commented:
> > > > a
> > > > > > > how so?
> > > > > > 
> > > > > > 
> > > > > > because actually you can insert reactor or antique? It's also op? Your logic is strange, if you think that speed is too much op, then don't buff it??? You just making op thing and nerf another which not related at all to balance it... It's strange.
> > > > > 
> > > > > 
> > > > > isnt the powerdraw too great to actually bring value? plus the antique is very rare and hard to find
> > > > 
> > > > 
> > > > move fast -> enter in maints -> wait 10-20 seconds -> move fast. It's more op than just power cage because you need recharge it with recharger when reactor it's just wait
> > > 
> > > 
> > > counter point being that you can barely use the weapons- if you miss it will be very punishing
> > 
> > 
> > No? I played on mech with reactor power cell and it works well
> 
> Also rinary i think the speed buff is needed considering its said to be ''lightly'' armored and right now it moves kind of slow

tbh gygax already the fastest mech, you just make it more op.

ScarletLightweaverSS14 commented:
> > > > > a
> > > > > > > > how so?
> > > > > > > 
> > > > > > > 
> > > > > > > because actually you can insert reactor or antique? It's also op? Your logic is strange, if you think that speed is too much op, then don't buff it??? You just making op thing and nerf another which not related at all to balance it... It's strange.
> > > > > > 
> > > > > > 
> > > > > > isnt the powerdraw too great to actually bring value? plus the antique is very rare and hard to find
> > > > > 
> > > > > 
> > > > > move fast -> enter in maints -> wait 10-20 seconds -> move fast. It's more op than just power cage because you need recharge it with recharger when reactor it's just wait
> > > > 
> > > > 
> > > > counter point being that you can barely use the weapons- if you miss it will be very punishing
> > > 
> > > 
> > > No? I played on mech with reactor power cell and it works well
> > 
> > 
> > Also rinary i think the speed buff is needed considering its said to be ''lightly'' armored and right now it moves kind of slow
> 
> tbh gygax already the fastest mech, you just make it more op.

no it isnt. the paddy is faster is it not

Rinary1 commented:
> > > > > > a
> > > > > > > > > how so?
> > > > > > > > 
> > > > > > > > 
> > > > > > > > because actually you can insert reactor or antique? It's also op? Your logic is strange, if you think that speed is too much op, then don't buff it??? You just making op thing and nerf another which not related at all to balance it... It's strange.
> > > > > > > 
> > > > > > > 
> > > > > > > isnt the powerdraw too great to actually bring value? plus the antique is very rare and hard to find
> > > > > > 
> > > > > > 
> > > > > > move fast -> enter in maints -> wait 10-20 seconds -> move fast. It's more op than just power cage because you need recharge it with recharger when reactor it's just wait
> > > > > 
> > > > > 
> > > > > counter point being that you can barely use the weapons- if you miss it will be very punishing
> > > > 
> > > > 
> > > > No? I played on mech with reactor power cell and it works well
> > > 
> > > 
> > > Also rinary i think the speed buff is needed considering its said to be ''lightly'' armored and right now it moves kind of slow
> > 
> > 
> > tbh gygax already the fastest mech, you just make it more op.
> 
> no it isnt. the paddy is faster is it not

no? If it is - it's strange

ScarletLightweaverSS14 commented:
<img width="588" height="368" alt="image" src="https://github.com/user-attachments/assets/fc5c8048-c199-45c4-8d68-41df3e18c05d" />
gygax

Rinary1 commented:
> > > > > > > a
> > > > > > > > > > how so?
> > > > > > > > > 
> > > > > > > > > 
> > > > > > > > > because actually you can insert reactor or antique? It's also op? Your logic is strange, if you think that speed is too much op, then don't buff it??? You just making op thing and nerf another which not related at all to balance it... It's strange.
> > > > > > > > 
> > > > > > > > 
> > > > > > > > isnt the powerdraw too great to actually bring value? plus the antique is very rare and hard to find
> > > > > > > 
> > > > > > > 
> > > > > > > move fast -> enter in maints -> wait 10-20 seconds -> move fast. It's more op than just power cage because you need recharge it with recharger when reactor it's just wait
> > > > > > 
> > > > > > 
> > > > > > counter point being that you can barely use the weapons- if you miss it will be very punishing
> > > > > 
> > > > > 
> > > > > No? I played on mech with reactor power cell and it works well
> > > > 
> > > > 
> > > > Also rinary i think the speed buff is needed considering its said to be ''lightly'' armored and right now it moves kind of slow
> > > 
> > > 
> > > tbh gygax already the fastest mech, you just make it more op.
> > 
> > 
> > no it isnt. the paddy is faster is it not
> 
> no? If it is - it's strange

Than just make it the same speed as paddy i think? Because paddy don't have such nerfs

ScarletLightweaverSS14 commented:
<img width="766" height="372" alt="image" src="https://github.com/user-attachments/assets/40b41754-7eb9-4be5-990a-1a25dcfe3248" />
paddy

ScarletLightweaverSS14 commented:
il remove it then @Rinary1

ScarletLightweaverSS14 committed:
changes per request

ScarletLightweaverSS14 committed:
Resolve merge conflict: confirm deletion of _StarLight/tags.yml

ScarletLightweaverSS14 committed:
Move MaulerArmor to _StarLight directory

ScarletLightweaverSS14 commented:
@Rinary1  done your requests

Rinary1 commented on Resources/Prototypes/Entities/Objects/Power/powercells.yml:
```diff
@@ -76,14 +76,9 @@
-  parent: PowerCellSmall
-  components:
-  - type: Sprite
+- type: entityte
```
```suggestion
- type: entity
```

ScarletLightweaverSS14 committed:
Fix prototype loading order for PowerCellTiny

Move Starlight power cells to base Entities directory to ensure BasePowerCell loads before PowerCellTiny tries to inherit from it. Fixes YAML linter crash.

Rinary1 commented on Resources/Prototypes/Entities/Objects/Devices/Electronics/mech.yml:
```diff
@@ -203,9 +203,6 @@
     - type: Sprite
       sprite: Objects/Misc/module.rsi
       state: mcontroller
-    - type: Tag
-      tags:
-      - GygaxTargetingControlModule
     - type: GuideHelp
```
???

Rinary1 commented on Resources/Prototypes/Entities/Objects/Power/powercells.yml:
```diff
@@ -109,11 +104,13 @@
   parent: BasePowerCell
   components:
+  - type: Tag
+    tags:
```
```suggestion
```

Rinary1 commented on Resources/Prototypes/Entities/Objects/Power/powercells.yml:
```diff
@@ -144,11 +141,13 @@
   parent: BasePowerCell
   components:
+  - type: Tag
+    tags:
```
```suggestion
```

Rinary1 commented on Resources/Prototypes/Entities/Objects/Power/powercells.yml:
```diff
@@ -184,6 +183,8 @@
   parent: BasePowerCell
   components:
+  - type: Tag
+    tags:
```
```suggestion
```

Rinary1 commented on Resources/Prototypes/Entities/Objects/Power/powercells.yml:
```diff
@@ -219,6 +220,8 @@
   suffix: Full
   components:
+  - type: Tag
+    tags:
```
```suggestion
```

Rinary1 commented on Resources/Prototypes/Entities/Objects/Power/powercells.yml:
```diff
@@ -255,6 +258,8 @@
   parent: BasePowerCell
   components:
+    - type: Tag
+      tags:
```
```suggestion
```

Rinary1 commented on Resources/Prototypes/Entities/Objects/Power/powercells.yml:
```diff
@@ -409,6 +426,8 @@
   parent: BasePowerCell
   components:
+  - type: Tag
+    tags:
```
```suggestion
```

Rinary1 commented on Resources/Prototypes/Entities/Objects/Specific/Mech/Weapons/Gun/combat.yml:
```diff
@@ -216,9 +216,9 @@
   - type: Gun
     fireRate: 1
-    selectedMode: SemiAuto
+    selectedMode: FullAuto
```
```suggestion
    selectedMode: FullAuto # Starlight-edit
```

Rinary1 commented on Resources/Prototypes/Entities/Objects/Specific/Mech/Weapons/Gun/combat.yml:
```diff
@@ -216,9 +216,9 @@
+    selectedMode: FullAuto
     availableModes:
-      - SemiAuto
+      - FullAuto
```
```suggestion
      - FullAuto # Starlight-edit
```

Rinary1 commented on Resources/Prototypes/Entities/Objects/Specific/Mech/Weapons/Gun/combat.yml:
```diff
@@ -240,7 +240,7 @@
     state: mecha_carbine
   - type: Gun
-    fireRate: 1.2
+    fireRate: 1.3
```
```suggestion
    fireRate: 1.3 # Starlight-edit
```

Rinary1 commented on Resources/Prototypes/Entities/Objects/Specific/Mech/Weapons/Gun/combat.yml:
```diff
@@ -265,12 +265,10 @@
-    burstCooldown: 1.2
-    selectedMode: Burst
+    fireRate: 3.2
+    selectedMode: FullAuto
     availableModes:
-      - Burst
+      - FullAuto
```
```suggestion
    # Starlight-start
    fireRate: 3.2
    selectedMode: FullAuto 
    availableModes:
      - FullAuto
    # Starlight-end
```

ScarletLightweaverSS14 committed:
Update Resources/Prototypes/Entities/Objects/Power/powercells.yml

Co-authored-by: Rinary <rinary.super@gmail.com>

ScarletLightweaverSS14 committed:
Update Resources/Prototypes/Entities/Objects/Power/powercells.yml

Co-authored-by: Rinary <rinary.super@gmail.com>

ScarletLightweaverSS14 committed:
Update Resources/Prototypes/Entities/Objects/Specific/Mech/Weapons/Gun/combat.yml

Co-authored-by: Rinary <rinary.super@gmail.com>

ScarletLightweaverSS14 committed:
Update Resources/Prototypes/Entities/Objects/Specific/Mech/Weapons/Gun/combat.yml

Co-authored-by: Rinary <rinary.super@gmail.com>

ScarletLightweaverSS14 committed:
Update Resources/Prototypes/Entities/Objects/Specific/Mech/Weapons/Gun/combat.yml

Co-authored-by: Rinary <rinary.super@gmail.com>

ScarletLightweaverSS14 committed:
Update Resources/Prototypes/Entities/Objects/Power/powercells.yml

Co-authored-by: Rinary <rinary.super@gmail.com>

ScarletLightweaverSS14 committed:
Add Starlight comments to Ultra AC-2 changes

ScarletLightweaverSS14 committed:
Restore GygaxTargetingControlModule tag

ScarletLightweaverSS14 committed:
Fix prototype loading order - rename to zzz_ prefix

powercells_starlight.yml was loading BEFORE powercells.yml alphabetically (underscore comes before period in ASCII). Rename to zzz_powercells_starlight.yml to ensure BasePowerCell is defined before PowerCellTiny tries to inherit from it.

ScarletLightweaverSS14 committed:
Trigger CI re-run for zzz_ prefix fix

Rinary1 commented on Resources/Prototypes/Entities/Objects/Devices/Electronics/mech.yml:
```diff
@@ -205,7 +205,7 @@
     - type: Tag
       tags:
-      - GygaxTargetingControlModule
+        - GygaxTargetingControlModule
```
```suggestion
      - GygaxTargetingControlModule
```

Rinary1 commented on Resources/Prototypes/Entities/Objects/Power/powercells.yml:
```diff
@@ -76,14 +76,9 @@
-      - PowerCellSmall
+      - PowerCageSmall
 
 - type: entity
-  id: PowerCellSmallPrinted
-  suffix: Empty
-  parent: PowerCellSmall
-  components:
-  - type: Sprite
     layers:
```
???

Rinary1 commented on Resources/Prototypes/tags.yml:
```diff
@@ -734,9 +734,6 @@
 - type: Tag
   id: GygaxPeripheralsControlModule
 
-- type: Tag
-  id: GygaxTargetingControlModule
-
```
???

ScarletLightweaverSS14 committed:
Update Resources/Prototypes/Entities/Objects/Devices/Electronics/mech.yml

Co-authored-by: Rinary <rinary.super@gmail.com>

ScarletLightweaverSS14 committed:
Restore accidentally deleted PowerCellSmallPrinted and GygaxTargetingControlModule

These were unintentionally removed during battery system cleanup.

ScarletLightweaverSS14 committed:
updates per rinary request

ScarletLightweaverSS14 commented:
done

ScarletLightweaverSS14 commented:
@Rinary1

rylandsuper commented:
i didn't even know power cell cages fit in mechs

ScarletLightweaverSS14 commented:
bump

ScarletLightweaverSS14 committed:
Code cleanup: Remove duplicate using statements and unnecessary dependency in MechSystem

- Removed duplicate 'using Content.Server.Body.Systems' statement
- Removed duplicate 'using Content.Shared.Whitelist' statement
- Removed unnecessary IEntityManager dependency (already provided by EntitySystem base class)

These changes improve code quality without affecting functionality.

ScarletLightweaverSS14 committed:
Merge upstream/starlight-dev into mech branch

ScarletLightweaverSS14 committed:
patch

ScarletLightweaverSS14 commented:
@Rinary1  how can i pinpoint the issues in the conflics. cant find em

Rinary1 committed:
Merge branch 'starlight-dev' of github.com:ss14Starlight/space-station-14 into mech

# Conflicts:
#	Content.Server/Mech/Systems/MechSystem.cs

Rinary1 commented on Resources/Prototypes/_StarLight/Entities/Objects/Specific/powercells.yml:
???

ScarletLightweaverSS14 replied:
how- what huh??? how do i fix this

Rinary1 commented on Resources/Prototypes/Entities/Objects/Specific/Mech/mechs.yml:
```diff
@@ -425,7 +425,7 @@
   - type: MovementSpeedModifier
     baseWalkSpeed: 2.5
-    baseSprintSpeed: 4.5
+    baseSprintSpeed: 4.6
```
this doesn't change anything tbh

ScarletLightweaverSS14 committed:
patch

ScarletLightweaverSS14 commented:
@Rinary1  how do i fix tje [Resources/Prototypes/Entities/Objects/Power/zzz_powercells_starlight.yml](https://github.com/ss14Starlight/space-station-14/pull/2217/files/f9c3cf49bcd0c52325d074c28983e1571d583996#diff-cdc3f877b2de202bfa8b5676347767fb0028ace177f512b9929903e596cc8e7d)

Rinary1 commented:
> @Rinary1  how do i fix tje [Resources/Prototypes/Entities/Objects/Power/zzz_powercells_starlight.yml](https://github.com/ss14Starlight/space-station-14/pull/2217/files/f9c3cf49bcd0c52325d074c28983e1571d583996#diff-cdc3f877b2de202bfa8b5676347767fb0028ace177f512b9929903e596cc8e7d)

Rename it back or revert commit where you changed it?

ScarletLightweaverSS14 committed:
Move powercells file back to _StarLight folder and remove _starlight suffix as requested by rinary

ScarletLightweaverSS14 committed:
Rename zzz_powercells_starlight.yml back to powercells_starlight.yml as requested by rinary

ScarletLightweaverSS14 commented:
> > @Rinary1  how do i fix tje [Resources/Prototypes/Entities/Objects/Power/zzz_powercells_starlight.yml](https://github.com/ss14Starlight/space-station-14/pull/2217/files/f9c3cf49bcd0c52325d074c28983e1571d583996#diff-cdc3f877b2de202bfa8b5676347767fb0028ace177f512b9929903e596cc8e7d)
> 
> Rename it back or revert commit where you changed it?

there?

Rinary1 commented:
> > > @Rinary1  how do i fix tje [Resources/Prototypes/Entities/Objects/Power/zzz_powercells_starlight.yml](https://github.com/ss14Starlight/space-station-14/pull/2217/files/f9c3cf49bcd0c52325d074c28983e1571d583996#diff-cdc3f877b2de202bfa8b5676347767fb0028ace177f512b9929903e596cc8e7d)
> > 
> > Rename it back or revert commit where you changed it?
> 
> there?

It's also moved into upstream folders + _starlight in name. So you need to move it back and remove _starlight

ScarletLightweaverSS14 committed:
Resolve merge conflict: keep powercells.yml in _StarLight folder as requested by rinary

ScarletLightweaverSS14 commented:
> > > > @Rinary1  how do i fix tje [Resources/Prototypes/Entities/Objects/Power/zzz_powercells_starlight.yml](https://github.com/ss14Starlight/space-station-14/pull/2217/files/f9c3cf49bcd0c52325d074c28983e1571d583996#diff-cdc3f877b2de202bfa8b5676347767fb0028ace177f512b9929903e596cc8e7d)
> > > 
> > > 
> > > Rename it back or revert commit where you changed it?
> > 
> > 
> > there?
> 
> It's also moved into upstream folders + _starlight in name. So you need to move it back and remove _starlight

donsies @Rinary1

ScarletLightweaverSS14 committed:
Merge branch 'starlight-dev' into mech

Rinary1 commented on Resources/Prototypes/_StarLight/Entities/Objects/Specific/powercells.yml:
it's anyway in another folder.

ScarletLightweaverSS14 replied:
what...
@Rinary1

ScarletLightweaverSS14 committed:
Remove duplicate powercells.yml from Specific folder as per review

ScarletLightweaverSS14 committed:
Merge branch 'starlight-dev' into mech

ScarletLightweaverSS14 commented:
we love No EntityPrototype found with id YellowSlimeExtractPowercell linter error

ScarletLightweaverSS14 commented:
@Rinary1

Rinary1 committed:
Merge branch 'starlight-dev' into mech

Rinary1 commented on Resources/Prototypes/_StarLight/Entities/Objects/Power/powercells.yml:
now you just removed it...

ScarletLightweaverSS14 replied:
ll the requested changes have already been completed correctly! The PowerCage tags are present in the main file, and the duplicate file has been removed. The PR is in the correct state.



do i dont get it

ScarletLightweaverSS14 replied:
@Rinary1

Rinary1 replied:
> ll the requested changes have already been completed correctly! The PowerCage tags are present in the main file, and the duplicate file has been removed. The PR is in the correct state.
> 
> 
> 
> do i dont get it

It's isn't a duplicate file or tags file, you just removed all _Starlight powercells like brig medic's defib cell.

ScarletLightweaverSS14 replied:
Oh
Will fix tmr

Rinary1 committed:
Merge branch 'starlight-dev' into mech

ScarletLightweaverSS14 committed:
added a deleted file

ScarletLightweaverSS14 committed:
Merge branch 'mech' of https://github.com/ScarletLightweaverSS14/Starlight-Chimes into mech

ScarletLightweaverSS14 commented:
@Rinary1  added it back under SL

ScarletLightweaverSS14 commented:
@Rinary1

Rinary1 committed:
move powercells back

Rinary1 committed:
Merge branch 'starlight-dev' of github.com:ss14Starlight/space-station-14 into mech

Rinary1 commented on Resources/Prototypes/Entities/Objects/Power/powercells.yml:
```diff
@@ -114,7 +114,7 @@
 - type: entity
   name: medium-capacity power cell
-  description: A rechargeable power cell. This is the popular and reliable version.
+  description: A rechargeable power cell. This is the kind of power cell you find in advanced machines.
```
```suggestion
  description: A rechargeable power cell. This is the kind of power cell you find in advanced machines. # Starlight-edit
```

Rinary1 commented on Resources/Prototypes/Entities/Objects/Power/powercells.yml:
```diff
@@ -149,7 +149,7 @@
 - type: entity
   name: high-capacity power cell
-  description: A rechargeable standardized power cell. This premium brand stores up to 50% more energy than the competition.
+  description: A rechargeable power cell. This premium brand stores up to 50% more energy than the competition.
```
```suggestion
  description: A rechargeable power cell. This premium brand stores up to 50% more energy than the competition. # Starlight-edit
```

Rinary1 commented on Resources/Prototypes/Entities/Objects/Power/powercells.yml:
```diff
@@ -305,6 +305,9 @@
   components:
+  - type: Tag
+    tags:
+      - PowerCage
```
Move this into `BasePowerCage` and add Starlight comments

ScarletLightweaverSS14 committed:
Update Resources/Prototypes/Entities/Objects/Power/powercells.yml

Co-authored-by: Rinary <rinary.super@gmail.com>

ScarletLightweaverSS14 committed:
Update Resources/Prototypes/Entities/Objects/Power/powercells.yml

Co-authored-by: Rinary <rinary.super@gmail.com>

ScarletLightweaverSS14 committed:
Merge branch 'mech' of https://github.com/ScarletLightweaverSS14/Starlight-Chimes into mech

ScarletLightweaverSS14 committed:
patch

ScarletLightweaverSS14 commented:
@Rinary1

Rinary1 committed:
AGAIN return powercells

Rinary1 committed:
Merge branch 'starlight-dev' of github.com:ss14Starlight/space-station-14 into mech

# Conflicts:
#	Resources/Prototypes/_StarLight/Entities/Objects/Power/powercells.yml
